entry.go

```diff
@@ -19,9 +26,12 @@ import (
 type EntryServiceInfo struct {
 	MaxPrice string `json:"maxPrice"`
 	ListenIP string `json:"ListenIP"`
+	Address  string `json:"address"`
```

Comment:
Why adding these two fields? EntryServiceInfo is for entry user to specify forward mode service details in config. It should not be related to reverse mode